README.md

```diff
@@ -84,8 +84,8 @@ An example assessment JSON can be found [here](src/assessment/sample_assessment.
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-------:|:--------:|
 | name | Page name in the format of `{assessment identifier}-{integer}-{descriptor}` (e.g. "RV0000-1-AutoForward"). | string | | yes |
-| capture_credentials | Indicates to GoPhish if the page will forward after an action. | boolean | | yes |
-| capture_passwords | Allows for capturing of user input, currently not used by PCA. | boolean | `False` | no |
+| capture_credentials | Capture all non-password input with GoPhish. Supports forwarding after submit action. | boolean | | yes |
+| capture_passwords | Capture password input by the user, but note that captured input is **currently stored in plain text.** | boolean | `False` | no |
```

Comment:
These are good improvements, though I would remove the word "currently" and add ", as of GoPhish version xxxx." to the end of that description.